ownership-chain/node/src/command.rs

```diff
@@ -1,33 +1,34 @@
-use std::net::SocketAddr;
+use std::{net::SocketAddr, sync::Arc};
 
+use cumulus_client_cli::generate_genesis_block;
 use cumulus_primitives_core::ParaId;
+use fc_db::kv::frontier_database_dir;
 use frame_benchmarking_cli::{BenchmarkCmd, SUBSTRATE_REFERENCE_HARDWARE};
 use laos_ownership_runtime::Block;
-use log::{info, warn};
-use polkadot_service::RococoChainSpec;
+use log::info;
+use parity_scale_codec::Encode;
 use sc_cli::{
 	ChainSpec, CliConfiguration, DefaultConfigurationValues, ImportParams, KeystoreParams,
 	NetworkParams, Result, SharedParams, SubstrateCli,
 };
-use sc_service::config::{BasePath, /* DatabaseSource, */ PrometheusConfig};
-use sp_runtime::traits::AccountIdConversion;
-// Frontier
-// use fc_db::frontier_database_dir;
+use sc_service::{
+	config::{BasePath, PrometheusConfig},
+	DatabaseSource, PartialComponents,
+};
+use sp_core::hexdisplay::HexDisplay;
+use sp_runtime::traits::{AccountIdConversion, Block as BlockT};
 
+#[cfg(feature = "try-runtime")]
+use crate::service::ParachainNativeExecutor;
 use crate::{
 	chain_spec,
 	cli::{Cli, RelayChainCli, Subcommand},
-	service::{self, /* db_config_dir, */ new_partial},
+	eth::db_config_dir,
+	service::new_partial,
 };
 
 fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	Ok(match id {
-		"arrakis" => Box::new(chain_spec::ChainSpec::from_json_bytes(
-			&include_bytes!("../../specs/arrakis-frontier.json")[..],
-		)?),
-		"caladan" => Box::new(chain_spec::ChainSpec::from_json_bytes(
-			&include_bytes!("../../specs/caladan-raw.json")[..],
-		)?),
 		"dev" => Box::new(chain_spec::development_config()),
 		"template-rococo" => Box::new(chain_spec::local_testnet_config()),
 		"" | "local" | "local-v" => Box::new(chain_spec::local_testnet_config()),
```

Comment:
what is local-v ?

Reply:
it was needed to recognise our chain as a chain with `unified account` in the zombienet config. it was a temporary fix that we did when we were migrating to unified accounts. Without this, we won't be able to run `zombienet` locally

Reply:
https://github.com/freeverseio/laos/pull/28